src/krux/pages/tiny_seed.py

```diff
@@ -179,7 +179,7 @@ def print_tiny_seed(self):
         pad_y = 8  # 2mm*8px
         self.ctx.display.clear()
         self.ctx.display.draw_hcentered_text(
-            t("Printing") + " ...", self.ctx.display.height() // 2
+            t("Printing") + "..", self.ctx.display.height() // 2
```

Comment:
I like this pattern (if it works in all languages "..")

Reply:
We have been using this 2 dots pattern since first version of Krux!

https://github.com/selfcustody/krux/blob/v22.03.0/i18n/translations/en-US.json

Reply:
I meant the `t("In the act of doing") + ".."` vs `t("In the act of doing..")` pattern.

Reply:
But Jeff's file (first version of Krux) doesn't distinguish between going to do or doing, see examples below:
```
Checking receive address %d for match..
Loading..
Shutting down..
Updating bootloader..
Upgrading firmware..
Backing up bootloader..
```

Reply:
Sorry! I saw what you mean now... it is the separation of the ponctuation from the word... I think we can't do that. I will change the above